test/quadquizaminos_web/live/power_up_test.exs

```diff
@@ -196,7 +196,7 @@ defmodule QuadquizaminosWeb.PowerUpTest do
 
       html = render_keydown(view, "keydown", %{"key" => " "})
 
-      refute html =~ "<i class=\"fas fa-arrows-alt\""
+      assert html =~ "<i class=\"fas fa-arrows-alt\""
```

Comment:
The same case applies here, the `move-block power-up` should be depleted once it's used . By asserting it's still there that means there is a loophole